src/components/drinking_age.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+ageVerify: function() {
```

Comment:
Doing it like this with jquery is fine for now and it works but it isn't the React way of doing it, it can all be done in React.

Reply:
Actually, how is this being used?

Reply:
We can actually delete this... started off with this code but ended up not using.